scripts/ort-reporter.sh

```diff
@@ -15,22 +15,60 @@ elif [[ "$ORT_DISABLE_SCANNER" = "false" && "$ORT_DISABLE_EVALUATOR" = "true" ]]
     ORT_RESULTS_INPUT_FILE=$ORT_RESULTS_SCANNER_FILE
 fi
 
+if [[ -s "$ORT_CONFIG_COPYRIGHT_GARBAGE_FILE" ]]; then
+    ORT_REPORTER_OPTIONS="--copyright-garbage-file $ORT_CONFIG_COPYRIGHT_GARBAGE_FILE"
+else
+    ORT_REPORTER_OPTIONS=""
+fi
+
+if [[ -d "$ORT_CONFIG_CUSTOM_LICENSE_TEXTS_DIR" ]]; then
+    ORT_REPORTER_OPTIONS="$ORT_REPORTER_OPTIONS --custom-license-texts-dir $ORT_CONFIG_CUSTOM_LICENSE_TEXTS_DIR"
+fi
+
+if [[ -s "$ORT_HOW_TO_FIX_TEXT_PROVIDER_FILE" ]]; then
+    ORT_REPORTER_OPTIONS="$ORT_REPORTER_OPTIONS --how-to-fix-text-provider-script $ORT_HOW_TO_FIX_TEXT_PROVIDER_FILE"
+fi
+
+if [[ -s "$ORT_CONFIG_LICENSE_CONFIGURATION_FILE" ]]; then
+    ORT_REPORTER_OPTIONS="$ORT_REPORTER_OPTIONS --license-classifications-file $ORT_CONFIG_LICENSE_CONFIGURATION_FILE"
+fi
+
+if [[ -d "$ORT_CONFIG_PACKAGE_CONFIGURATION_DIR" ]]; then
+    ORT_REPORTER_OPTIONS="$ORT_REPORTER_OPTIONS --package-configuration-dir $ORT_CONFIG_PACKAGE_CONFIGURATION_DIR"
+elif [[ -s "$ORT_CONFIG_PACKAGE_CONFIGURATION_FILE" ]]; then
+    ORT_REPORTER_OPTIONS="$ORT_REPORTER_OPTIONS --package-configuration-file $ORT_CONFIG_PACKAGE_CONFIGURATION_FILE"
+elif [[ -d "$ORT_CONFIG/packages" ]]; then
+    # Use legacy named package configuration directory if present.
+    ORT_REPORTER_OPTIONS="$ORT_REPORTER_OPTIONS --package-configuration-dir $ORT_CONFIG/packages"
+    ORT_CONFIG_PACKAGE_CONFIGURATION_DIR=$ORT_CONFIG/packages
+fi
+
+if [[ -s "$ORT_CONFIG_RESOLUTIONS_FILE" ]]; then
+    ORT_REPORTER_OPTIONS="$ORT_REPORTER_OPTIONS --resolutions-file $ORT_CONFIG_RESOLUTIONS_FILE"
+fi
+
+if [[ $ORT_REPORT_FORMATS =~ "EvaluatedModel" ]]; then
+   ORT_REPORTER_OPTIONS="$ORT_REPORTER_OPTIONS -O EvaluatedModel=output.file.formats=json"
+fi
+
+if [[ $ORT_REPORT_FORMATS =~ "NoticeTemplate" ]]; then
+   ORT_REPORTER_OPTIONS="$ORT_REPORTER_OPTIONS -O NoticeTemplate=project-types-as-packages=\"SpdxDocumentFile\""
+   ORT_REPORTER_OPTIONS="$ORT_REPORTER_OPTIONS -O NoticeTemplate=template.path=$ORT_CONFIG_NOTICE_TEMPLATE_PATHS"
+fi
+
+# FIXME: Below statement fails if SW_NAME contains a space
+# if [[ $ORT_REPORT_FORMATS =~ "SpdxDocument" ]]; then
+#    ORT_REPORTER_OPTIONS="$ORT_REPORTER_OPTIONS -O SpdxDocument=document.name='${SW_NAME}' \ -O SpdxDocument=output.file.formats=json,yaml"
```

Comment:
Have you considered using either the default document name, or using a name which is fully independent of the scan parameters, like for the other reports?

Reply:
Using the SW name is something that comes from ORT user input, who actually requested to move away from using default document name to the SW name. I think I can fix the FIXME in future commit but simply ran out of time now, would like to leave it 'as is'